libra_toolbox/neutronics/neutron_source.py

```diff
@@ -0,0 +1,78 @@
+# building the MIT-VaultLab neutron generator
+# angular and energy distribution
+
+from pathlib import Path
+from collections.abc import Iterable
+import pandas as pd
+import numpy as np
+import h5py
+import openmc
+
+
+def A325_generator_diamond(center=(0, 0, 0), reference_uvw=(0, 0, 1)) -> Iterable[openmc.Source]:
```

Comment:
`openmc.Source` is not a valid type. It is a function.
The type of the object returned by this function is `openmc.IndependentSource`

```suggestion
def A325_generator_diamond(center=(0, 0, 0), reference_uvw=(0, 0, 1)) -> Iterable[openmc.IndependentSource]:
```